### ISSUE TITLE
how to get buggy line number for each file?

### ISSUE THREAD

**Issue by SamraMehboob** [codeflaws/codeflaws#10]
Hi there,
Can you please let me know how to get buggy line number from each file. Do I need to use fault localization technique for this purpose or is there any other simpler way for codeflaws data. I mean to extract buggy-fixed line pair for each file.
Any help regarding this is much appreciated.
Thank you.

**Comment by stan6** [comment]
Hi, you can just do a diff between the two versions (buggy and correct).

**Comment by SamraMehboob** [comment]
Thank you for help, Do I need to take diff based on AST e:g using gumTree tool.

**Comment by stan6** [comment]
No. Unix diff will work. AST diff will not show the line difference as it will show the edit script.

**Comment by SamraMehboob** [comment]
Alright thanks. I'll try with it.
